app/viewmodels/PermissionToStartUnloading.scala

```diff
@@ -91,4 +91,6 @@ final case class PermissionToStartUnloading(
     printListOfItems &&
       consignorOne.isEmpty &&
       goodsItems.toList.flatMap(_.consignor).nonEmpty
+
+  val totalNumberOfPackages: String = numberOfPackages.fold("")(_.toString)
```

Comment:
Should this be an empty string or 0?

Reply:
@MatthewAlexanderBeck It should be an empty string [That is how they show in legacy System]